### DISCUSSION TITLE
Enable dates to work with disabled days in PrimeReact calendar

### DISCUSSION THREAD

**Discussion by sdwillbrand** [primefaces/community#1059] [PrimeReact] [answered]
Hey everybody,
we are currently with the PrimeReact calendar and we stumbled upon a small problem. While disabling dates we realized we wanted to enable some dates. It does not seem to work right now. Can one work with both ```disabledDays``` and ```enabledDates```?
Does anybody here have experience working with the calendar?

Cheers

**Comment by melloware** [comment]
Yes you should be able to use both.   Can you explain more what you are trying to do?  Here is a sandbox you can start with: https://codesandbox.io/p/sandbox/primereact-test-forked-cjgj6d?file=%2Fsrc%2Findex.js%3A20%2C11

**Reply by sdwillbrand** [reply]
Is your sandbox private? I am not able to see it.

**Reply by melloware** [reply]
OK i just made it public.  Code Sandbox has been making a lot of unwanted changes recently trying to monetize their app.  Can you try again?

**Reply by sdwillbrand** [reply]
It is working now, thank you!
We would like to use ```disabledDays``` with ```enabledDates```. 
[Example](https://codesandbox.io/p/sandbox/primereact-test-forked-cjy7lg)
Here we want to disable e.g Sunday to Wednesday but some specific days, for instance holidays, should be enabled. In our example we wanted to enable the 19.02.2024 but it is on a Monday and Mondays are disabled. Is it possible to make it work?

**Reply by melloware** [reply, marked as answer]
Got it can you please open a bug on GitHub Issues I think I know the issue. The enabledDates needs to be evaluated LAST after all the disables date stuff

**Reply by sdwillbrand** [reply]
I opened an issue! Thank you for your support!

**Reply by melloware** [reply]
OK i can't get to your sandbox.
